frontend/test/pytest/test_compiler.py

```diff
@@ -136,6 +138,56 @@ def test_attempts_to_get_inexistent_intermediate_file(self):
         with pytest.raises(ValueError, match="pass .* not found."):
             compiler.get_output_of("inexistent-file")
 
+    def test_runtime_error(self):
+        """Test that an exception is emitted when the runtime raises a C++ exception."""
+
+        class CompileCXXException:
+            """Class that overrides the program to be compiled."""
+
+            _executable = "clang++"
```

Comment:
Would this test fail for systems not having  `clang` installed?

Reply:
```suggestion
            _executable = "cc"
```